backend/events/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+from .models import Event
+
+
+class EventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        # fields = ('title', 'description', 'start_time', 'end_time', 'event_image',)
```

Comment:
Let's flip this back and delete the unused line.

Reply:
done